create-api-diff.sh

```diff
@@ -31,8 +31,8 @@ then
    # Install additional features: Xtext and dependent
    eclipse/eclipse -data eclipse/.director-ws -consolelog -noSplash -clean \
    -application org.eclipse.equinox.p2.director \
-   -metadataRepository http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.20.0/,http://download.eclipse.org/releases/2019-12,http://download.eclipse.org/lsp4j/updates/releases/,https://download.eclipse.org/tools/orbit/downloads/2019-12 \
-   -artifactRepository http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.20.0/,http://download.eclipse.org/releases/2019-12,http://download.eclipse.org/lsp4j/updates/releases/,https://download.eclipse.org/tools/orbit/downloads/2019-12 \
+   -metadataRepository http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.21.0/,http://download.eclipse.org/releases/2019-12,http://download.eclipse.org/lsp4j/updates/releases/,https://download.eclipse.org/tools/orbit/downloads/2019-12 \
```

Comment:
wonder if we should use 2020-03 here (and below)

Reply:
Looks like it was changed in the last release also. Didn't see that, because why would GitHub render the whole line on a wide screen?

Will adapt